### PR TITLE
Ft Authentication

### PR DESCRIPTION
**Pull request by offiongbassey**
User 
authentication
Authentication Completed.

### CODE REVIEW

File: src/controllers/auth.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { jwtVerification } from "../helpers/jwtVerification";
+
+export const createUser = async (req, res) => {
+    try {
+        const { name, username, email, phone, gender, password } = req.body;
+        const salt = await bcrypt.genSalt(10);
+        const hashedPassword = await bcrypt.hash(password, salt);
+        let userName;
+        //generating unique username
+        if(username === ""){
```
Comment (eleezs) [inline src/controllers/auth.js:17]: make this a function of its own,

File: src/helpers/redis_connect.js
Comment (eleezs) [inline src/helpers/redis_connect.js:1]: move it to the server config folder

**Review by eleezs — CHANGES_REQUESTED** [review]

File: src/helpers/redis_connect.js
Comment (offiongbassey) [reply src/helpers/redis_connect.js:1]: Okay. I have done that.

File: src/controllers/auth.js (hunk 1 above)
Comment (offiongbassey) [reply src/controllers/auth.js:17]: Okay. I have done that.

**Review by eleezs — APPROVED** [review]

**Pull request merged**